tools/nl/embeddings/data/preindex/sdg/duplicate_names.csv

```diff
@@ -1,24 +1,58 @@
 PreferredSV,DroppedSV,DuplicateName
+dc/topic/sdg_11,dc/topic/sdg_1.1,sdg 11
+dc/topic/sdg_11,dc/topic/sdg_1.1,sdg11
+dc/topic/sdg_11.1,dc/topic/sdg_1.1.1,sdg 111
+dc/topic/sdg_11.1,dc/topic/sdg_1.1.1,sdg111
 dc/topic/sdg_1.5.1,dc/topic/sdg_11.5.1,"Number of deaths, missing persons and directly affected persons attributed to disasters per 100K population"
 dc/topic/sdg_1.5.2,dc/topic/sdg_11.5.2,Direct economic loss attributed to disasters in relation to global gross domestic product (GDP)
 dc/topic/sdg_1.5.3,dc/topic/sdg_11.b.1,Number of countries that adopt and implement national disaster risk reduction strategies in line with the Sendai Framework for Disaster Risk Reduction 2015-2030
 dc/topic/sdg_1.5.4,dc/topic/sdg_11.b.2,Proportion of local governments that adopt and implement local disaster risk reduction strategies in line with national disaster risk reduction strategies
+dc/topic/sdg_12,dc/topic/sdg_1.2,sdg 12
+dc/topic/sdg_12,dc/topic/sdg_1.2,sdg12
+dc/topic/sdg_12.1,dc/topic/sdg_1.2.1,sdg 121
+dc/topic/sdg_12.1,dc/topic/sdg_1.2.1,sdg121
+dc/topic/sdg_12.2,dc/topic/sdg_1.2.2,sdg 122
+dc/topic/sdg_12.2,dc/topic/sdg_1.2.2,sdg122
 dc/topic/sdg_8.4.1,dc/topic/sdg_12.2.1,"Material footprint, material footprint per capita, and material footprint per GDP"
 dc/topic/sdg_8.4.2,dc/topic/sdg_12.2.2,"Domestic material consumption, domestic material consumption per capita, and domestic material consumption per GDP"
 dc/topic/sdg_4.7.1,dc/topic/sdg_12.8.1,(b) curricula
 dc/topic/sdg_4.7.1,dc/topic/sdg_12.8.1,(c) teacher education
 dc/topic/sdg_4.7.1,dc/topic/sdg_12.8.1,Extent to which (i) global citizenship education and (ii) education for sustainable development are mainstreamed in (a) national education policies
 dc/topic/sdg_4.7.1,dc/topic/sdg_12.8.1,and (d) student assessment
 dc/topic/sdg_7.b.1,dc/topic/sdg_12.a.1,Installed renewable energy-generating capacity in developing countries (in watts per capita)
+dc/topic/sdg_13,dc/topic/sdg_1.3,sdg 13
+dc/topic/sdg_13,dc/topic/sdg_1.3,sdg13
+dc/topic/sdg_13.1,dc/topic/sdg_1.3.1,sdg 131
+dc/topic/sdg_13.1,dc/topic/sdg_1.3.1,sdg131
 dc/topic/sdg_1.5.1,dc/topic/sdg_13.1.1,"Number of deaths, missing persons and directly affected persons attributed to disasters per 100K population"
 dc/topic/sdg_1.5.3,dc/topic/sdg_13.1.2,Number of countries that adopt and implement national disaster risk reduction strategies in line with the Sendai Framework for Disaster Risk Reduction 2015-2030
 dc/topic/sdg_1.5.4,dc/topic/sdg_13.1.3,Proportion of local governments that adopt and implement local disaster risk reduction strategies in line with national disaster risk reduction strategies

```

Comment:
nit: missing texts for some of these?

Reply:
No, these are just the duplicates so the texts are ok here. What's more interesting is that some texts are exactly the same which we should update at some point. But that's Ok for now because it seems they refer to the same thing.